tools/labs/Makefile

```diff
@@ -32,8 +32,16 @@ copy: $(YOCTO_IMAGE)
 	rmdir $(TEMPDIR)
 
 docker-docs:
-	cd docs && docker-compose build
-	cd docs && docker-compose run docs-build bash -c "cd /linux/tools/labs && make docs"
+	cd docker && docker-compose build docs-build
+	cd docker && UID=$(shell id -u) GID=$(shell id -g) docker-compose run docs-build bash -c "cd /linux/tools/labs && make docs"
+
+docker-kernel:
+	cd docker && docker-compose build kernel-build
+	cd docker && docker-compose up -d kernel-build
```

Comment:
@tavip , what are the `UID` and `GID` variable definitions useful for? Should they be added to the `docker-kernel` and `stop-docker-kernel` target as well?

Reply:
I have made changes so that now I add a user that has the same UID and GID as the docker build calling user, so there won't be any ID mixing.